packages/esm-patient-allergies-app/src/allergies/allergies-form/allergy-form.workspace.tsx

```diff
@@ -42,20 +42,21 @@ import { type Allergy, useAllergies } from '../allergy-intolerance.resource';
 import { AllergenType } from '../../types';
 import styles from './allergy-form.scss';
 
-const allergyFormSchema = z.object({
-  allergen: z
-    .object({
-      uuid: z.string(),
-      display: z.string(),
-      type: z.string(),
-    })
-    .required(),
-  nonCodedAllergen: z.string().optional(),
-  allergicReactions: z.array(z.string().optional()),
-  nonCodedAllergicReaction: z.string().optional(),
-  severityOfWorstReaction: z.string(),
-  comment: z.string().optional(),
-});
+const allergyFormSchema = (t: TFunction) =>
+  z.object({
+    allergen: z
+      .object({
+        uuid: z.string(),
+        display: z.string(),
+        type: z.string(),
+      })
+      .required(),
+    nonCodedAllergen: z.string().optional(),
+    allergicReactions: z.array(z.string().optional()),
+    nonCodedAllergicReaction: z.string().optional(),
+    severityOfWorstReaction: z.string(),
+    comment: z.string().optional(),
```

Comment:
```suggestion
    comments: z.string().optional(),
```

Let's rename this control to `comments`.